util/src/memorydb.rs

```diff
@@ -181,7 +181,10 @@ impl HashDB for MemoryDB {
 	}
 
 	fn keys(&self) -> HashMap<H256, i32> {
-		self.data.iter().filter_map(|(k, v)| if v.1 != 0 {Some((k.clone(), v.1))} else {None}).collect()
+		self.data.iter()
```

Comment:
Why remove a `filter_map`? Clippy produces a warning on `.filter().map()` sequence

Reply:
ah, ok

I just found it more readable than this long one-liner

Reply:
fixed